app/src/main/java/edu/uc/it3048/mypokedex/MainActivity.kt

```diff
@@ -7,7 +7,6 @@ import android.widget.ImageButton
 import edu.uc.it3048.mypokedex.ui.main.MainFragment
 
 class MainActivity : AppCompatActivity() {
-
```

Comment:
Removed white space between class declaration and first line of class. This is done in a few places in the project. Seen in source here: https://kotlinlang.org/docs/reference/coding-conventions.html